daemon/daemon.go

```diff
@@ -734,6 +734,17 @@ func NewDaemon(dp datapath.Datapath) (*Daemon, *endpointRestoreState, error) {
 	// Must be done before calling policy.NewPolicyRepository() below.
 	identity.InitWellKnownIdentities()
 
+	epMgr := endpointmanager.NewEndpointManager(&endpointsynchronizer.EndpointSynchronizer{})
+
+	// Cleanup flannel on exit
+	cleanupFuncs.Add(func() {
+		if option.Config.FlannelUninstallOnExit {
```

Comment:
nit: Could probably swap this and the line above to only install the run function in this case, rather than always running it.

Reply:
Fixed in latest commit!